packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -100,3 +106,8 @@
 .block-editor-panel-color-gradient-settings__dropdown {
 	width: 100%;
 }
+
+.block-editor-color-gradient-control__tabs {
+	border-bottom: 1px solid $gray-300;
+	margin-bottom: $grid-unit-10;
+}
```

Comment:
Overriding styles this way introduces fragmentation in the way a component looks across the app, and makes this code less resilient to future design changes.

It would be great if we could use the component as it comes out of the box (or alternatively, we could look at updating the component to fit the new design requirements).

Reply:
As far as I know all implementations of `TabPanel` in core append a bottom border, so we should explore adding it to the component, and removing these styles holistically.

Reply:
That sounds like a good plan, at least in principle!

Reply:
From a purely design perspective I find the tabs to look a little awkward with no stroke beneath them. But thinking about backwards compatibility I suppose we could add a prop to the component that creates a 'borderless' appearance?

Reply:
On one hand, we could
- add an `isBorderless` prop (with a default value of `true`, so that it doesn't introduce a visual change to the component)
- replace all instances where we're adding a a border with a style override by setting `isBorderless` to `false`

But I wonder if this is the right approach:
- I'm not a massive fan of props that are very visually descriptive, as they expose an implementation detail of the design (in this case, showing a border or not) which may not make sense in future iterations
- this pattern of adding props and introduce variants to our components is often a way to mask consistency issues

I'd prefer a scenario in which:
- we investigate how the component is used across Gutenberg (and potentially plugins)
- we assess if the component's design needs updating. In this case, relatively to the bottom border, we should assess if there can be only one version of the component (with or without border) — which would be our preferred option — or if it's absolutely necessary to expose both variants.
- we update the component accordingly and remove style overrides across Gutenberg.